files/default/neutron-ha-tool.py

```diff
@@ -561,13 +563,14 @@ def replicate_dhcp(qclient, noop=False):
 
 
 def migrate_l3_routers_from_agent(qclient, agent, targets,
-                                  noop, wait_for_router, delete_namespace):
+                                  noop, wait_for_router, delete_namespace,
+                                  agent_picker):
```

Comment:
The order of the argument is not really consistent between all methods. Could you tweak things a bit?